src/gui/screen_bed_leveling.cpp

```diff
@@ -0,0 +1,85 @@
+#include "screen_bed_leveling.hpp"
+#include "log.h"
+#include "ScreenHandler.hpp"
+#include "gui.hpp"
+
+screen_bed_leveling_t::screen_bed_leveling_t()
+    : AddSuperWindow<screen_t>()
+    , m_statusMsg(this, { 10, 128, 220, 80 }, is_multiline::no)
+    , m_progressBar(this, Rect16(10, 70, GuiDefaults::RectScreen.Width() - 2 * 10, 16))
+    , header(this)
+    , footer(this) {
+    header.SetIcon(IDR_PNG_heatbed_16px);
+    header.SetText(_("BED LEVELING"));
+    this->m_statusMsg.SetText(_(""));
```

Comment:
Text will be initialized as empty, but if you want use directly stringview insted of translating empty string.